mne/minimum_norm/psf_ctf.py

```diff
@@ -1,407 +0,0 @@
-# Authors: Olaf Hauk <olaf.hauk@mrc-cbu.cam.ac.uk>
-#          Alexandre Gramfort <alexandre.gramfort@inria.fr>
-#
-# License: BSD (3-clause)
-
-from copy import deepcopy
-
-import numpy as np
-from scipy import linalg
-
-from ..io.constants import FIFF
-from ..io.pick import pick_channels
-from ..utils import logger, verbose, _check_option, deprecated
-from ..forward import convert_forward_solution
-from ..evoked import EvokedArray
-from ..source_estimate import SourceEstimate
-from .inverse import _subject_from_inverse
-from . import apply_inverse
-
-
-def _prepare_info(inverse_operator):
-    """Get a usable dict."""
-    # in order to convert sub-leadfield matrix to evoked data type (pretending
-    # it's an epoch, see in loop below), uses 'info' from inverse solution
-    # because this has all the correct projector information
-    info = deepcopy(inverse_operator['info'])
-    info['sfreq'] = 1000.  # necessary
-    info['projs'] = inverse_operator['projs']
-    return info
-
-
-def _pick_leadfield(leadfield, forward, ch_names):
-    """Pick out correct lead field components."""
-    # NB must pick from fwd['sol']['row_names'], not ['info']['ch_names'],
-    # because ['sol']['data'] may be ordered differently from functional data
-    picks_fwd = pick_channels(forward['sol']['row_names'], ch_names)
-    return leadfield[picks_fwd]
-
-
-@deprecated('point_spread_function is deprecated and will be removed in 0.21;'
-            'please get_point_spread instead.')
-@verbose
-def point_spread_function(inverse_operator, forward, labels, method='dSPM',
```

Comment:
I don't think these are supposed to be removed yet, we are on the 0.20 development cycle and these should stay until 0.21